articles/managed-services/man-patching-faq.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Patching as a Service FAQs
+description: Frequently asked questions for Patching as a Service
+services: managed-services
+author: Steve Dixon
+reviewer:
+lastreviewed: 08/10/2020 15:17:17
+toc_rootlink: Managed IT Operations
+toc_sub1: Patching as a Service
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Patching as a Service FAQs
+toc_fullpath: Managed IT Operations/Patching as a Service/man-patching-faq.md
+toc_mdlink: man-patching-faq.md
+---
+
+# Patching as a Service FAQs
+
+## Service
+
+### What is the service?
+
+As customer solutions start to expand beyond their own data centres into embracing the benefits of leveraging multi-cloud, the day-to-day management and control of operating system (OS) patching across various technology stacks and locations become highly complex and resource intensive. Patching as a Service helps to reduce these challenges, whilst increasing your level of trust and confidence in the security health of any operating systems deployed as part of your solution stack.
+
+With Patching as a Service, we provide routine OS patch management for compute instances deployed across UKCloud's multi-cloud. It helps you accelerate business value by removing the burden of basic OS hygiene, letting you focus your attention above the operating system.
+
+### What technologies do you use?
+
+Built-upon the capabilities of our [Managed Monitoring as a Service](https://docs.ukcloud.com/articles/managed-services/man-monitoring-sco.html), UKCloud uses a combination of patches released directly by the vendor coupled with [Ivanti Security Controls](https://www.ivanti.com/products/security-controls), a market leading enterprise patch management and control solution, as the key technologies to deliver Patching as a Service.
+
+This service also leverages the benefits of AIOps (see below).
+
+### Do I get access to a dashboard or reports?
+
+As it is assumed UKCloud will manage all patching aspects as part of this service, there is currently no dashboard or reporting features for customers to access.
+
+### How do I get alerted on events?
+
+All alerting will be performed via a support ticket or any special escalation process agreed between UKCloud and the customer.
+
+### How does UKCloud provide urgent maintenance notifications and incident reports?
+
+We envisage that there should not be a scenario which prevents UKCloud in meeting the availability KPI defined within the [Service Scope](https://docs.ukcloud.com/articles/managed-services/man-patching-sco.html) for this service. Current Service Status reports are published on the [Status Page](https://status.ukcloud.com/). You can view past incident reports on the UKCloud Portal.
```

Comment:
You can see past incidents on the Service Status page as well. Not sure that we need to point customers separately to the Portal

Reply:
Good spot! Now corrected